src/main/java/com/nhnacademy/hexashoppingmallservice/controller/category/CategoryController.java

```diff
@@ -142,11 +118,24 @@ public ResponseEntity<Void> deleteCategory(@PathVariable Long categoryId,
     }
 
     @PostMapping("/{categoryId}/books/{bookId}")
-    public ResponseEntity<Void> getAllBooksByCategoryId(@PathVariable Long categoryId, @PathVariable Long bookId) {
+    public ResponseEntity<Void> insertBook(@PathVariable Long categoryId, @PathVariable Long bookId) {
         categoryService.insertBook(categoryId, bookId);
         return new ResponseEntity<>(HttpStatus.NO_CONTENT);
     }
 
+    @GetMapping("/{categoryId}")
```

Comment:
BookController에서 이미 카테고리 아이디를 이용한 도서 목록 반환 로직이 있는데, 여기에 유사한 로직을 추가하신 이유가 무엇인가요?
```suggestion
    @GetMapping("/{categoryId}")
```

Reply:
페이징 처리를 안한 모든 도서 목록들을 가져오기 위해서 추가했습니다.